test/useForm.test.tsx

```diff
@@ -445,42 +445,62 @@ describe('useForm', () => {
     )
     expect(result.current.fields.details.description.touched).toEqual(false)
   })
+})
 
-  it('should pass current state value in validation rule function', async () => {
-    const validator: ValidationRule<string, Widget> = (value, state) => {
-      if (state.details.description === 'PictureRequired' && !value) {
-        return 'Picture is required'
-      }
-      return
+describe('useForm validation rules', () => {
```

Comment:
We had a nice unit test that covered the "run validation" when `setValue` is called (run validation when a single field changes), I just added an extra test to cover the case where we run `validate` instead (run validation across the whole form).